components/layout/fragment_tree/box_fragment.rs

```diff
@@ -88,7 +88,10 @@ pub(crate) struct BoxFragment {
 
     block_margins_collapsed_with_children: Option<Box<CollapsedBlockMargins>>,
 
-    /// The scrollable overflow of this box fragment.
+    /// The scrollable overflow of this box fragment in the same coordiante system as
+    /// [`Self::content_rect`] ie a rectangle within the parent fragment's content
```

Comment:
I have always seen it as "i.e.", not "ie".

Reply:
"i.e." is technically correct, but "ie" is also used informally. It's common if a bit cheeky.